README.md

```diff
@@ -66,7 +68,7 @@ curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-osx
 To install on linux run:
 
 ```bash
-curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-linux-amd64 -o opkssh; chmod +x opkssh
+curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-linux-$(uname -p) -o opkssh; chmod +x opkssh
```

Comment:
This doesn't work on amd64 (x86_64)

![image](https://github.com/user-attachments/assets/08e9600b-af4f-46ca-bdce-9a5293156ec9)

Maybe just change this to say for amd64 do X and for arm64 do X